src/axis.js

```diff
@@ -147,6 +152,27 @@ c3_axis_internal_fn.splitTickText = function (d, scale) {
 
     return split(splitted, tickText + "");
 };
+c3_axis_internal_fn.ellipsify = function(splitted, max) {
+    if (splitted.length <= max) {
+        return splitted;
+    }
+
+    var ellipsified = splitted.slice(0, max);
+    var remaining = 3;
+    for (var i = max-1 ; i >= 0 ; i--) {
+        var available = ellipsified[i].length;
+
+        ellipsified[i] = ellipsified[i].substr(0, available-remaining).padEnd(available, '.');
```

Comment:
`padEnd` doesn't seem available on IE11 which c3 still supports. Probably we need util function for this.